ui/src/components/app/footerLinks/Impressum.js

```diff
@@ -1,13 +1,19 @@
 import React, { Component } from "react";
 
 class Impressum extends Component {
+
+    componentDidMount(){
```

Comment:
Not sure if this is common within the UI (@NJaku01) but I'd prefer the simpler `componentMounted()` over using "did".

Reply:
I didn't know this function was part of ReactJS - ignore my comment: https://reactjs.org/docs/react-component.html#componentdidmount